origin-graphql/src/contracts.js

```diff
@@ -32,6 +32,8 @@ if (typeof window !== 'undefined') {
 
 const Configs = {
   mainnet: {
+    // provider:
```

Comment:
We don't have to do this in this PR, but could we start using envkey for all these config variables ?